### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by Maksym-Perepelytsia**
### Hooli-style Popup

[Demo](https://maksym-perepelytsia.github.io/html-css-popup/) |
[Code base](https://github.com/Maksym-Perepelytsia/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Maksym-Perepelytsia** [comment]
I changed some CSS code because the 'More Cats' focus styles weren't working.

File: submissions/Maksym-Perepelytsia/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    <li class="header__menu__item">
+                        <a href="#" class="menu__link">News</a>
+                    </li>
+                    <li class="header__menu__item">
+                        <a href="#" class="menu__link">Contact</a>
+                    </li>
+                    <li class="header__menu__item popup-toggle">
+                        <input type="checkbox" id="popup-checkbox">
+                        <label for="popup-checkbox" class="popup-label">
+                            <img src="img/popup-button.png" alt="popup icon" width="24" height="24">
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/index.html:31]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

File: submissions/Maksym-Perepelytsia/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    opacity: 1;
+    background: linear-gradient(90deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 50%, rgba(0,0,0,0) 100%);
+}   
+
+.popup-toggle {
+    border-radius: 50%;
+}
+
+#popup-checkbox,
+#more-cats-checkbox{
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/style.css:86]: You have class names on this elements, why do you styling by id? It is better to use class names for styling, and ids for js

File: submissions/Maksym-Perepelytsia/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    display: block;
+    padding: 5px;
+}
+
+.popup__menu__item a:hover {
+    border-radius:5px;
+    box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.8);
+}
+
+.popup__menu__item img {
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/style.css:143]: Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

File: submissions/Maksym-Perepelytsia/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+.footer a {
+    color: #000;
+}
+
+h1 {
+    text-align: center;
+    line-height: 1.5em;
+}
+
+@media screen and (max-width:380px){
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/style.css:191]:
```suggestion
@media screen and (max-width:380px) {
```
You forget space here

File: submissions/Maksym-Perepelytsia/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+    .popup__wrapper{
+        left:15px;
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/style.css:193]:
```suggestion
        left: 15px;
```
You forget space here

File: submissions/Maksym-Perepelytsia/html-css-popup/style.css [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+}
+
+.more-cats-btn {
+    display: block;
+    margin: 10px 5px 5px;
+    padding: 5px;
+    border-radius:5px;
+    background-color: #d3d3d3;
+    text-align: center;
+    cursor:pointer;
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/style.css:154]:
```suggestion
    cursor: pointer;
```
You forget space here

File: submissions/Maksym-Perepelytsia/html-css-popup/style.css [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+}   
+
+.popup-toggle {
+    border-radius: 50%;
+}
+
+#popup-checkbox,
+#more-cats-checkbox{
+    appearance: none;
+    position:absolute;
```
Comment (HannaSyn) [inline submissions/Maksym-Perepelytsia/html-css-popup/style.css:88]:
```suggestion
    position: absolute;
```
You forget space here

**Review by HannaSyn — CHANGES_REQUESTED** [review]
You did good job!
Please, see my comments below

**Comment by Maksym-Perepelytsia** [comment]
@HannaSyn Thank you for your feedback. Please, review my latest edits according to your comments.

**Review by Maksym-Perepelytsia — COMMENTED** [review]
Alt attributes are now empty.
In the CSS file, id selectors changed to class selectors.
Styling by tags was replaced with styling by classes.
And I fixed the missing spaces.

**Comment by Maksym-Perepelytsia** [comment]
Thank you very much.
Have a nice day 🙂

**Pull request merged**